src/redux/slices/userSlice.ts

```diff
@@ -6,11 +6,15 @@ import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 interface UserState {
   id?: DSNPUserId;
   walletType: wallet.WalletType;
+  displayedProfileId?: DSNPUserId;
 }
 
 const initialState: UserState = {
   id: session.hasSession() ? session.loadSession()?.id : undefined,
   walletType: session.loadSession()?.walletType ?? wallet.WalletType.NONE,
+  displayedProfileId: session.hasSession()
```

Comment:
This shouldn't be necessary. An undefined `displayedProfileId` should behave the same as when the `displayedProfileId === id`.